Library/Formula/miller.rb

```diff
@@ -0,0 +1,33 @@
+class Miller < Formula
+  desc "Like sed, awk, cut, join & sort for name-indexed data such as CSV"
+  homepage "https://github.com/johnkerl/miller"
+  url "https://github.com/johnkerl/miller/releases/download/v2.2.1/mlr-2.2.1.tar.gz"
+  sha256 "2e85b0a2beb295174a4346f98dec79573394064c1fc2f004fe6816feefb0ddbe"
+
+  head do
+    url "https://github.com/johnkerl/miller.git"
+
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+    depends_on "libtool" => :build
+  end
+
+  def install
+    system "autoreconf", "-fvi" if build.head?
+    system "./configure", "--prefix=#{prefix}", "--disable-silent-rules",
+                          "--disable-dependency-tracking"
+    system "make"
+    system "make", "check"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.csv").write <<-EOS.undent
+      a,b,c
+      1,2,3
+      4,5,6
+    EOS
+    output = pipe_output("#{bin}/mlr --csv --rs lf cut -f a test.csv")
```

Comment:
According to the latest docs, the `--rs` argument is supposed to be ignored for `--csv` argument. There may be a bug and it's not being ignored, but if that gets fixed, the tests could be made more resilient by using `--csvlite` or putting CRLF line endings in the CSV fixtures.